web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -230,7 +230,8 @@
                       <v-list-item
                         v-for="el in item.services"
                         :key="el.name"
-                        :href="el.url"
+                        @click="el.isPublicNeuroglancer ? redirectNeuroglancerUrl(item) : null"
+                        :href="!el.isPublicNeuroglancer ? el.url : null"
```

Comment:
as for embargoed -- we are providing URLs unconditionally ATM but they would not even work for embargoed. Filed more generic:

- https://github.com/dandi/dandi-archive/issues/2065
- https://github.com/dandi/dandi-archive/issues/2066